consensus/core/src/events/mod.rs

```diff
@@ -14,7 +14,7 @@ pub enum ConsensusEvent {
 
 #[derive(Debug, Clone, Default)]
 pub struct VirtualChangeSetEvent {
-    pub utxo_diff: Arc<UtxoDiff>,
+    pub selected_parent_utxo_diff: Arc<UtxoDiff>, //Note: this differs from the Consensus [`VirtualState`]'s [`UtxoDiff`] and represents the utxo-state of the selected parent.
```

Comment:
This is the other way around: `VirtualState.utxo_diff` is the virtual diff from its selected parent. This represents the diff from the previous virtual UTXO set